src/translations/es.translation

```diff
@@ -6508,6 +6508,14 @@ Nota: La ficha Configuración de NPSP es visible en la aplicación Nonprofit Suc
     <customLabels>
         <label>Al establecer el modelo de cuenta en &quot;Cuenta de Hogar&quot;, se requieren que las reglas de hogares se establezcan en &quot;Sin contactos&quot;. Al utilizar este modelo, no hay necesidad de tener un objeto adicional Hogar.</label>
         <name>stgValidationHHAccountHHRules</name>
+    </customLabels>
```

Comment:
Assuming these translations were copied over from another repository, they have to be placed in each `translation` file in the correct spot alphabetically.